coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -4,7 +4,7 @@ set oem_id="ec2"
 
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
-set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable"
+set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable console=tty1 console=ttyS0 xen_blkfront.max_indirect_segments=256"
```

Comment:
the `xen_blkfront.max_indirect_segments=256` setting was recommended for sc1/st1 (hdd) volumes, but i see gp2 (ssd) being used on at least hot